src/config.ts

```diff
@@ -52,7 +52,7 @@ export const TOKEN_MIN_AMOUNT = 0.00000001
 export const DOLLAR_MIN_AMOUNT_ACCOUNT_FUNDED = 0.01
 // The number of seconds before the sender can reclaim the payment.
 export const ESCROW_PAYMENT_EXPIRY_SECONDS = 1 // The contract doesn't allow 0 seconds.
-export const DEFAULT_TESTNET = Config.DEFAULT_TESTNET
+export const DEFAULT_TESTNET = Config.DEFAULT_TESTNET!
```

Comment:
Is it safe to use `!` here and below? Wonder if we should check for existence and throw if not found or at least have some defaults?

Reply:
Ah yes it could be a bit better 👍